code/modules/clothing/glasses/_glasses.dm

```diff
@@ -119,17 +120,24 @@
 
 /obj/item/clothing/glasses/night
 	name = "night vision goggles"
-	desc = "You can totally see in the dark now!"
+	desc = "You can totally see in the dark now! Just don't look too closely at bright lights. This lacks any flash correction."
 	icon_state = "night"
 	item_state = "glasses"
 	darkness_view = 8
+	flash_protect = -2
```

Comment:
This is a bit extreme, no?

Reply:
At -2 in a hardsuit/welder mask you should be able to survive flashes but flashbangs and welders will get you.

Also, don't weld with NVG's. That's probably the best advice I have. This is primarily to combat it's use while fighting, not while doing construction work. If you're doing that, you have plenty of time to swap goggles or you need to make a choice.